sys/bloom/bloom.c

```diff
@@ -35,7 +35,7 @@ bloom_t *bloom_new(size_t size, size_t num_hashes, ...)
     }
 
     /* Allocate Bloom array */
-    if (!(bloom->a = calloc(ROUND(size), sizeof(char)))) {
+    if (!(bloom->a = calloc(sizeof(uint8_t), ROUND(size)))) {
```

Comment:
```c
void *calloc(size_t nmemb, size_t size);
```
>The  calloc()  function allocates memory for an array of nmemb elements of size bytes each and returns a pointer to the allocated memory.

I'd say the original argument order is correct, but there is something else causing scan-build to warn here.